src/Sarif/EnumeratedArtifact.cs

```diff
@@ -87,50 +91,28 @@ public byte[] Bytes
                 this.Stream = FileSystem.FileOpenRead(Uri.LocalPath);
             }
 
-            if (Stream.CanSeek)
-            {
-                RetrieveDataFromSeekableStream();
-            }
-            else
-            {
-                RetrieveDataFromNonSeekableStream();
-            }
+            RetrieveDataFromStream();
 
             this.Stream = null;
 
             return (this.contents, this.bytes);
         }
 
-        private void RetrieveDataFromNonSeekableStream()
+        private void RetrieveDataFromStream()
         {
-            bool isText;
-
-            this.bytes = new byte[Stream.Length];
-            int length = this.Stream.Read(this.bytes, 0, this.bytes.Length);
-            isText = FileEncoding.IsTextualData(this.bytes, 0, length);
+            // Reset to beginning of stream in case caller neglected to do so.
+            TryRewindStream();
```

Comment:
I would love to remove this call.  It can't be necessary, right?  But I sort of inherited it from the Seekable variant.  Would love to hear I can delete.

Reply:
Sure, let's zap it.